doc: app_dev: bootloaders: sysbuild: Fix variable usage (#16933)
nrfconnect/sdk-nrf: doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst

      Escaped quotations avoid malformed-string warnings from Kconfig.

> The path of the key must be an absolute path, though the escaped ``${APPLICATION_CONFIG_DIR}`` CMake variable (by replacing the ``$`` with ``\$``) can be used to get the path of the application configuration directory to use keys relative to this directory.

Reviewer: ```suggestion
The path of the key must be an absolute path. 
To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``). 
```
tried to make it flow better, please double-check the meaning

Author: Fixed

      Escaped quotations avoid malformed-string warnings from Kconfig.

The path of the key must be an absolute path.

> The To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``).

Reviewer: ```suggestion
To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``.
```

Author: Silly mistake that, fixed

Reviewer: the leftover bracket wasn't removed - can we get rid of that?

Author: Now should be fixed
